modules/miscutil/lib/mailutils.py

```diff
@@ -24,24 +24,24 @@
 
 __revision__ = "$Id$"
 
+import os
+import re
 import sys
+from cStringIO import StringIO
```

Comment:
if you are using `six` for Python 3 compatibility then use check also [`six.StringIO`](https://pythonhosted.org/six/#six.StringIO).